### PR TITLE
swayws: init at unstable-2022-03-10

### PR DESCRIPTION
**Pull request by AtilaSaraiva**
###### Description of changes

This is a sway workspace tool which allows easy moving of workspaces to and from outputs.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/sway/ws.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    rev = "514f3c664439cf2c11bb9096c7e1d3b8c0b898a2";
+    sha256 = "sha256-vUnbn79v08riYLMBI8BxeBPpe/pHOWlraG7QAaohw3s=";
+  };
+
+  cargoSha256 = "sha256-PvKpcTewajvbzUHPssBahWVcAQB3V/aMmOJ/wA0Nrv4=";
+
+  nativeBuildInputs = [];
+  buildInputs = [ ];
+
+  # has not tests
```
Comment (b12f) [inline pkgs/applications/window-managers/sway/ws.nix:19]:
```suggestion
  # swayws does not have any tests
```

**Comment by teutat3s** [comment]
Result of `nixpkgs-review pr 188078` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>swayws</li>
  </ul>
</details>

File: pkgs/applications/window-managers/sway/ws.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    owner = "w0lff";
+    repo = pname;
+    rev = "514f3c664439cf2c11bb9096c7e1d3b8c0b898a2";
+    sha256 = "sha256-vUnbn79v08riYLMBI8BxeBPpe/pHOWlraG7QAaohw3s=";
+  };
+
+  cargoSha256 = "sha256-PvKpcTewajvbzUHPssBahWVcAQB3V/aMmOJ/wA0Nrv4=";
+
+  nativeBuildInputs = [];
+  buildInputs = [ ];
```
Comment (teutat3s) [inline pkgs/applications/window-managers/sway/ws.nix:17]: You can remove these empty arrays, they're the default.

**Review by teutat3s — COMMENTED** [review]
Thank you for adding this great package. Looking forward to seeing this merged.

File: pkgs/applications/window-managers/sway/ws.nix (hunk 2 above)
Comment (AtilaSaraiva) [reply pkgs/applications/window-managers/sway/ws.nix]: Thanks for the review! I removed them now.

File: pkgs/applications/window-managers/sway/ws.nix (hunk 1 above)
Comment (AtilaSaraiva) [reply pkgs/applications/window-managers/sway/ws.nix]: Thanks for the review! Indeed seems better that way, done!

**Review by AndersonTorres — APPROVED** [review]

**Pull request merged**